.github/workflows/openqa.yml

```diff
@@ -0,0 +1,42 @@
+---
+name: Run a basic openQA test
+# yamllint disable-line rule:truthy
+on:
+  pull_request_target:
+  # Note how this runs on:pull_request_target and not on:pull_request!
+  # The difference is that this runs always with the context of the master
+  # branch. This is necessary to allow accessing the API credential secrets.
+  workflow_dispatch:
+env:
+  OPENQA_HOST: ${{ secrets.OPENQA_URL }}
+  OPENQA_API_KEY: ${{ secrets.OPENQA_API_KEY }}
+  OPENQA_API_SECRET: ${{ secrets.OPENQA_API_SECRET }}
```

Comment:
That's prevented by the use of "pull_request_target" which is also why you don't see a checkmark before we merge this yet. See https://github.com/os-autoinst/os-autoinst-distri-opensuse/pull/18182#issuecomment-1822549621

Reply:
`workflow_dispatch` makes the check run manually.

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#workflow_dispatch

 `pull_request_target` event docs say:

> This event runs in the context of the base of the pull request, rather than in the context of the merge commit, as the pull_request event does. This prevents execution of unsafe code from the head of the pull request that could alter your repository or steal any secrets you use in your workflow. This event allows your workflow to do things like label or comment on pull requests from forks. Avoid using this event if you need to build or run code from the pull request.

https://docs.github.com/en/actions/using-workflows/events-that-trigger-workflows#pull_request_target

I'd strongly suggest doing some tests with a dummy secret to validate those behaviors

Reply:
We already do the same in the example and openQA test distributions which access the exact same set of secrets (same GitHub organization). So I'm confident it'll work here as well.

Reply:
discussed with @foursixnine in person we are ok with it as is